packages/calypso-build/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Changelog
 
-## master
+## master (6.5.0)
+
+- Added `--esm` and `--cjs` options to `copy-assets` and `transpile` to do only one kind of
+  the build (ESM or CJS) instead of both that are done by default
 
 ## 6.4.0
 
-- Removed the exceptions for the `import/no-extraneous-dependencies` eslint rule for '*.md.jsx' and '*.md.js' files
+- Removed the exceptions for the `import/no-extraneous-dependencies` eslint rule for '_.md.jsx' and '_.md.js' files
```

Comment:
Looks like this was autofixed by prettier wrong, those should be `*.md.js`

Reply:
Nice catch, fixing in #47179.